apps/test/unit/lab2/views/components/versionHistory/VersionHistoryButtonTest.tsx

```diff
@@ -36,6 +34,16 @@ const ownedChannel: Channel = {
   updatedAt: '',
 };
 
+// Mock this component to speed up tests.
```

Comment:
Nice! Just adding a [link](https://codedotorg.slack.com/archives/C08AMQ869QX/p1740683882034219?thread_ts=1740683699.090069&cid=C08AMQ869QX) to related Slack discussion which was helpful to read.